styles.css

```diff
@@ -377,3 +395,251 @@ form textarea {
+  }
+
+  .introduction h2 {
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .i-text {
+    font-size: 20px;
+    line-height: 28px;
+    padding: 0.83% 18.5% 0 18.5%;
+    height: 6.8%;
+  }
+
+  .i-h4 {
+    font-size: 16px;
+    line-height: 24px;
+    padding: 0.3% 50% 0.83% 18.5%;
+  }
+
+  .social-media {
+    padding: 0 50% 20% 18.5%;
+  }
+
+  .projects {
+    display: grid;
+    margin: 4.99% 0;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .desktop {
+    display: block;
+  }
+
+  article {
+    display: flex;
+    margin: 4.931% 9.862%;
+  }
+
+  .order-0 {
+    flex: none;
+    flex-grow: 2;
+    order: 2;
+  }
+
+  .order-1 {
+    flex: none;
+    flex-grow: 0;
+    order: 3;
+  }
+
+  .order-2 {
+    flex: none;
+    flex-grow: 0;
+    order: 0;
+  }
+
+  .order-3 {
+    flex: none;
+    flex-grow: 0;
+    order: 1;
+  }
+
+  article img {
+    width: 60%;
+    padding: 2.076% 1.903% 2.076% 2.076%;
+  }
+
+  article h3 {
+    font-size: 40px;
+    line-height: 52px;
+    padding: 4.83% 4.06%;
+  }
+
+  .pr-details {
+    padding: 2.076% 1.903% 2.336% 0;
+  }
+
+  article h5 {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  article h6 {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+  }
+
+  article ul {
+    align-items: center;
+    height: 7.135%;
+    padding: 0 4.06%;
+  }
+
+  article p {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    padding: 4.83% 4.06%;
+  }
+
+  article ul li img {
+    padding: 0;
+  }
+
+  button {
+    margin: 8.076% 4.06%;
+  }
+
+  .about div {
+    border-radius: 0 7% 0 0;
+    width: 100%;
+  }
+
+  .about > .container {
+    display: flex;
+  }
+
+  .about-intro {
+    display: flex;
+    flex-direction: column;
+    width: 42.15%;
+  }
+
+  .about-h2 {
+    padding: 25.353% 0 0 25.353%;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .about-text {
+    color: #344563;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-size: 20px;
+    line-height: 28px;
+    padding: 1% 1% 0 25.353%;
+  }
+
+  .about-h4 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    padding: 2% 1% 0 25.353%;
+  }
+
+  .about-media {
+    padding: 2% 1% 0 27%;
+  }
+
+  .about-button {
+    margin: 8.076% 46.06% 24.839% 25.353%;
+  }
+
+  .about-skills {
+    padding: 11.094% 11.094% 19% 2%;
+  }
+
+  .lang-list {
+    display: flex;
+    flex-direction: row;
+    padding: 0 21px 0 0;
+  }
+
+  .lang-list li {
+    display: flex;
+    flex-direction: column;
+    margin-right: 10px;
+  }
+
+  .contact-form {
+    background-image: url(assets/images/contact-form-background-image.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 100% 100%;
+  }
+
```

Comment:
Your design looks so good and responsive too. Good job at that.

Kindly align the "Get in touch" button to the center as given by the design so we'll be as close to the design as possible.

Sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now. You'll also get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible.

Reply:
Thank you!